tests/models/longt5/test_modeling_longt5.py

```diff
@@ -584,6 +590,10 @@ def test_model_from_pretrained(self):
             model = LongT5Model.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @unittest.skipIf(
+        is_torch_available_and_torch_less_than_1_11,
```

Comment:
I originally tried to use `not is_torch_available or is_torch_less_than_1_11`, but it always complains `is_torch_less_than_1_11` is not defined (when `torch`  is not installed).

It looks like the `or` evaluation doesn't early return in this usage with unittest (it works in usual python statement)

Reply:
It does return early but the variable is just not defined if `torch` is not installed. I think you can just set `is_torch_less_than_1_11=None` in an `else` block at line 46 of this file and then it will work.

Reply:
Well, my bad, I used `is_torch_available`, which should be `is_torch_available()`.

`is_torch_available` is a function, so it evaluates to `True` and the statement continues to evaluate the 2nd term.

With `is_torch_available()`, it works now (even without the `else` block you mentioned) - but I can add it if you think it's better to be explicit.

Reply:
That works for me!